charts/alfresco-connector-ms365/values.yaml

```diff
@@ -0,0 +1,44 @@
+global:
+  alfrescoRegistryPullSecrets: quay-registry-secret
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+replicaCount: 2
+nodeSelector: {}
+image:
+  repository: quay.io/alfresco/alfresco-ooi-service
+  tag: 1.1.3
+  pullPolicy: IfNotPresent
+  internalPort: 9095
+service:
+  name: ms365-service
+  type: ClusterIP
+  externalPort: 80
+ingress:
+  path: /ms365-service
+  tls: []
+  extraAnnotations:
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+podSecurityContext:
+  runAsNonRoot: true
+  runAsUser: 33006
+resources:
+  requests:
+    cpu: "0.25"
+    memory: "1000Mi"
+  limits:
+    cpu: "2"
+    memory: "1000Mi"
+environment:
+  JAVA_OPTS: " -Dalfresco.base-url=http://acs-alfresco-cs-repository:80"
```

Comment:
this will be overridden by the parent chart, but as a default doesn't make a lot of sense

since `alfresco.base-url` is a mandatory property for the service to work properly, if there are no better ways to define it other than JAVA_OPTS, I would make available a new value for this chart and then in the template we concatenate it in JAVA_OPTS, it would be much more cleaner and avoid user mistakenly overwriting it